src/newSeed/baseStage.scss

```diff
@@ -1,66 +1,100 @@
 @import "styles/colors";
 // This styles are common to all the stages components
 // Theme for the stages
-@mixin stageTheme {
-  * {
-    font-size: 1rem;
+.infoContainer {
+  font-size: 1rem;
+  font-family: Inter;
+  background-color: $color-black-3;
+  border: 1px solid $color-darkpink-2;
+  border-radius: 1.5em;
+  padding: 2em;
+  min-width: 40rem;
+  .title {
+    font-size: 1.625em;
+    color: $color-white;
+    line-height: 2em;
+    font-weight: bold;
   }
-  .infoContainer {
-    background-color: $color-black-3;
-    border: 1px solid $color-darkpink-2;
-    border-radius: 1.5em;
-    padding: 2em;
-    min-width: 40rem;
-    .title {
-      font-size: 1.625em;
-      color: $color-white;
-      line-height: 2em;
-      font-weight: bold;
-    }
-    .subtext {
-      line-height: 1.5em;
-      font-size: 1em;
-      color: #dcdae5;
-      margin-bottom: 5em;
-    }
-    .infoInput {
-      margin-bottom: 2em;
-    }
-    .buttonContainer {
-      display: flex;
-      flex-flow: row wrap;
-      align-items: center;
-      column-gap: 3em;
-      font-weight: bold;
-      margin-top: 5em;
-
-      .cancel {
-        color: $color-orange;
-        font-family: Inter;
-      }
-    }
-    label {
-      display: block;
-      font-size: 1.1em;
-      line-height: 2em;
-      color: $color-white;
-    }
-    input {
-      width: 26em;
-      height: 2.7em;
-      border: 1px solid $color-darkpink-2;
+  .subtext {
+    line-height: 1.5em;
+    font-size: 1em;
+    color: #dcdae5;
+    margin-bottom: 5em;
+  }
+  .infoInput {
+    margin-bottom: 2em;
+  }
+  .buttonContainer {
+    display: flex;
+    flex-flow: row wrap;
+    column-gap: 3em;
+    font-weight: bold;
```

Comment:
remove this.  the button styles are already set globally.  if it is for ".cancel" then put it there.

Reply:
This is for the container holding the buttons and not for the buttons themselves. It is needed.

Reply:
what is in the container except buttons?

Reply:
Nothing but buttons but there are 2 buttons and the container is for me to style how they are positioned in the container

Reply:
position is fine, but this comment is on line 30 which is font-weight, which you don't need here

Reply:
Aha, didn't catch that